### PR TITLE
Test. #1250

### PR DESCRIPTION
**Pull request by Learath2**
Just want artifacts for #1250

### CODE REVIEW

File: src/base/system.c [hunk 1]
```diff
@@ -262,13 +262,26 @@ void mem_zero(void *block,unsigned size)
...
 	if(flags == IOFLAG_WRITE)
-		return (IOHANDLE)fopen(filename, "wb");
+		modes = "wb";
 	if(flags == IOFLAG_APPEND)
-		return (IOHANDLE)fopen(filename, "ab");
-	return 0x0;
+		modes = "ab";
+
+	errno = 0;
+	result = (IOHANDLE)fopen(filename, modes);
```
Comment (msiglreith) [inline src/base/system.c]: Usage of potentially uninitialized value `modes`

File: src/base/system.c [hunk 2]
```diff
@@ -262,13 +262,26 @@ void mem_zero(void *block,unsigned size)
...
+	if(!result){
```
Comment (msiglreith) [inline src/base/system.c]: nit: formatting

**Comment by Learath2** [comment]
It's just a test build for #1250. Not thinking of ever merging this :)

**Comment by Learath2** [comment]
Eh, might as well fix the uninitialized use.

**Comment by heinrich5991** [comment]
Outputting useful error messages is also an area in which Teeworlds/DDNet could improve.

**Comment by Learath2** [comment]
Discuss in #1317.

**Pull request closed without merging**